utils.ts

```diff
@@ -174,7 +181,8 @@ const filterOutDuplicateStops = (
     // duplicate
     return arePointsRoughlyEqual(
       feature.geometry.coordinates,
-      otherFeature.geometry.coordinates
+      otherFeature.geometry.coordinates,
+      parseInt(LOCATION_COMPARISON_COORDIATE_ACCURACY || '')
```

Comment:
Can you throw a `LOCATION_COMPARISON_COORDINATE_ACCURACY &&` in front of this so that we don't evaluate the `parseInt` if we don't need to? Running it on a blank string feels a little icky